eduvpn/ui/stats.py

```diff
@@ -0,0 +1,139 @@
+import os
+import logging
+import time
+from datetime import timedelta
+
+from ..nm import get_iface, get_ipv4, get_ipv6, get_timestamp
+
+logger = logging.getLogger(__name__)
+
+LINUX_STATS_FILE = "/sys/class/net/{iface}/statistics/{filename}"
+
+
+class NetworkStats():
+    def __init__(self, interval):
+        self.iface = None
+        self.interval = interval
```

Comment:
This doesn't seem to be used.

Reply:
ah yeah sorry the interval was for the /s data rate, that is not needed any more. Thanks